src/Components/ArticlesPageComponents/ArticlesList.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { useEffect } from "react";
+import { getArticles } from "../../api";
+import axios from "axios";
+import ArticleCard from "./ArticleCard";
+
+const ArticlesList =()=>{
+
+const [articlesList, setArticlesList] = useState([])
+const [isLoading, setIsLoading] = useState(true)
+
+const articlesApi = axios.create({
+    baseURL: "https://v2-nc-news-backend-project-northcoders.onrender.com"
+})
+useEffect(()=>{
```

Comment:
Don't forget to set isLoading back to true first, in case the component renders again after it's been set to false